appeals/web/src/server/appeals/appeal-details/interested-party-comments/__tests__/interested-party-comments.test.js

```diff
@@ -182,4 +183,34 @@ describe('interested-party-comments', () => {
 			expect(invalidMessage?.textContent?.trim()).toBe('No invalid comments');
 		});
 	});
+
+	describe('GET /review-comment with data', () => {
+		beforeEach(() => {
+			nock('http://test/').get('/appeals/2/reps/5').reply(200, interestedPartyCommentForReview);
+		});
+
+		it('should render review comment page with the provided comment details', async () => {
+			const response = await request.get(`${baseUrl}/2/interested-party-comments/5/review`);
+
+			expect(response.statusCode).toBe(200);
+
+			const elementInnerHtml = parseHtml(response.text).innerHTML;
+			expect(elementInnerHtml).toMatchSnapshot();
```

Comment:
There should be at least one check for specific content using `toContains` as a safeguard to ensure the correct page is being rendered, just to reduce the burden of reading through snapshots